src/output.cpp

```diff
@@ -1566,12 +1566,12 @@ std::map<size_t, inclusive_rectangle<point>> draw_tabs( const catacurses::window
     std::map<size_t, inclusive_rectangle<point>> tab_map;
 
     int width = getmaxx( w );
-    for( int i = 0; i < width; i++ ) {
```

Comment:
Note to self: Check if I didn't break something with this in another use of this function

Reply:
I don't seem to have broken anything with that.